app/src/main/java/com/arctouch/codechallenge/extensions/GlideExtensions.kt

```diff
@@ -0,0 +1,24 @@
+package com.arctouch.codechallenge.extensions
+
+import android.annotation.SuppressLint
+import android.widget.ImageView
+import com.arctouch.codechallenge.MainApplication
+import com.arctouch.codechallenge.R
+import com.bumptech.glide.Glide
+import com.bumptech.glide.RequestBuilder
+import com.bumptech.glide.request.RequestOptions
+
+@SuppressLint("CheckResult")
+@JvmOverloads
+fun ImageView.loadUrl(url: String, hasPlaceholder: Boolean? = false){
```

Comment:
Nullability is one of the most useful Kotlin features.
Could you explain why `hasPlaceholder` is nullable?
Can you think a way you could avoid making it nullable?

Reply:
The idea here was to set a default boolean argument, but analyzing it the nullability could be removed as this would never be null and if no parameter is passed to it, the default value is false. So, removing the nullability would be fine for this case, just keeping it like placeHolder: Boolean = false